packages/@webex/plugin-meetings/src/metrics/index.ts

```diff
@@ -83,27 +19,6 @@ class Metrics {
    */
   constructor() {
     if (!Metrics.instance) {
```

Comment:
hmmm this old code is effectively a singleton - how did this work without hitting the same issue you had?

Reply:
We suspect because it was in the same module, and not in a different plugin